scripts/installs/install_rails_service.bat

```diff
@@ -0,0 +1,3 @@
+copy C:\Vagrant\resources\rails_server\start_rails_server.bat "C:\Program Files\Rails_Server"
+schtasks /create /tn "rails" /tr "\"cmd.exe\" /c \"C:\Program Files\Rails_Server\start_rails_server.bat\"" /sc onstart /NP /ru "SYSTEM"
```

Comment:
It never starts the service, it's weird. It would just say "service started successfully", but port 3000 is never up. It's weird. I never figured out how to make it work, so I switched to schtasks.